app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,85 @@
+#modalOverlay {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.